src/custom-types.json

```diff
@@ -78,7 +78,11 @@
   },
   "application/javascript": {
     "charset": "UTF-8",
-    "compressible": true
+    "compressible": true,
+    "extensions": ["mjs"],
+    "sources": [
+      "https://github.com/nodejs/node-eps/blob/master/002-es-modules.md"
```

Comment:
I don't see the mime listed on that page, though there is a link to another page saying .mjs should be type application/javascript+module

Can the page be updated to clarify what the mime is or link to a different source that does?